interface/ceed-fortran.c

```diff
@@ -343,7 +370,7 @@ void fCeedElemRestrictionApplyBlock(int *elemr, int *block, int *tmode,
     FORTRAN_NAME(ceedelemrestrictiongetmultiplicity,CEEDELEMRESTRICTIONGETMULTIPLICITY)
 void fCeedElemRestrictionGetMultiplicity(int *elemr, int *mult, int *err) {
   *err = CeedElemRestrictionGetMultiplicity(CeedElemRestriction_dict[*elemr],
-                                            CeedVector_dict[*mult]);
+         CeedVector_dict[*mult]);
```

Comment:
Is also this line supposed to be accepted according to make style or keep as was? (Asking because I'm going through the same changes in my branch)

Reply:
I think we want it to be like this.

Reply:
fixed

Reply:
Even though it is different from other function calls?

Reply:
The goal is always to have make style return no changes on master. I have to figure out what is strange in the density current header, but that's not part of this PR (ideally).

I just ran make style to fix my changes consistent with our style. Sometimes some PRs get through without make style, and I am catching some of those issues here.

Reply:
I understand the goal. But this is not in the density current header and I'm trying to understand why we should accept this change, if the previous version was still in our style guidelines, no? The original line did not exceed 90 characters and the line as is now would be just different from all other function call styles, where we indent at the parenthesis.

Reply:
Make style wouldn't have changed it if it was consistent. The function calls indent 4 spaces or at the parentheses, depending upon how deep the parentheses are. To see this, look at the QFunction portion of this header.